### PR TITLE
Update Demo app to work with Sensor 0.2.0

### PR DESCRIPTION
**Pull request by marcinreliga**
I updated the Demo app to work with Sensor 0.2.0. In the process I also applied a couple of other changes:

- updated pods to the latest
- fixed tests
- replaced dependency on Alamofire with simple native code

### CODE REVIEW

File: DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  Created by Marcin Religa on 30/6/22.
+//  Copyright © 2022 Intelligent Apps GmbH. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIImageView {
+    func imageFromServerURL(_ URLString: String, placeHolder: UIImage?) {
+        self.image = nil
```
Comment (stbix) [inline DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift:14]: Shouldn't the placeholder go here?

File: DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    self.image = placeHolder
+                }
+                return
+            }
+            DispatchQueue.main.async {
+                if let data = data {
+                    if let downloadedImage = UIImage(data: data) {
+                        self.image = downloadedImage
+                    }
+                }
```
Comment (stbix) [inline DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift:36]: The two if-statements could be joined into one.

**Review by stbix — APPROVED** [review]

**Review by ghGarnik — APPROVED** [review]

File: DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift (hunk 1 above)
Comment (marcinreliga) [reply DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift:14]: It depends if we want to always show the placeholder and then change it to a loaded image, or if we want to show placeholder only if loading the image fails.

I'd say for this demo it probably doesn't matter much.

File: DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift (hunk 2 above)
Comment (marcinreliga) [reply DemoApp/LoginDemo/Utility/UIImageView+imageFromServerURL.swift:36]: Yes, done.

**Pull request merged**